src/version.rs

```diff
@@ -45,22 +43,11 @@ pub fn long_ver() -> String {
 pub fn print() {
     eprintln!("tremor version: {}", long_ver().as_str());
     eprintln!("tremor instance: {}", instance!());
-    let (version_n, version_s) = get_rdkafka_version();
-    eprintln!("rd_kafka version: 0x{:08x}, {}", version_n, version_s);
 }
 
 /// Logs tremor and librdkafka version.
 pub fn log() {
     info!("tremor version: {}", long_ver().as_str());
-    let (version_n, version_s) = get_rdkafka_version();
```

Comment:
do we really want to get rid of the kafka version, that has been  super helpful printout

Reply:
Good catch, that is an artifact of my attempts to move connectors to their own module, will re-add